java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -499,4 +501,201 @@ public void testIgnoresPrefixMatchFlag() throws IOException {
             "class TestClassPartialMatch { }")
         .doTest();
   }
+
+  /**
+   * [https://github.com/uber/piranha/issues/141]
+   *
+   * <p>Test to check that Java enum constants are removed when they have a constructor fields that
+   * matches the flag value
+   *
+   * <p>Also tests flags that use unqualified enum constant usages, and enum method calls
+   */
+  @Test
+  public void testRemoveEnumFieldValueMatchUntreated() {
```

Comment:
We "recently" put some overdue work to partition `CorePiranhaTest` into multiple test classes, which is both better for readability and allows running our tests in parallel with gradle.

Could we get all tests related to enum constant support into their own test suite class? (I know that's a small amount of boilerplate, but we really don't want to end up with 2-3k  `CorePiranhaTest` again 😅